pyproject.toml

```diff
@@ -51,6 +51,7 @@ classifiers = [
 testing = [
     'coverage',
     'pyyaml',
+    'packaging'
```

Comment:
I have initially added the `packaging` dependency. I just wanted to verify that that resolves the issue. I am not committed to this resolution. We may still drop the one test which requires it, in which case the dependency will be removed.